pkg/controller/spd/spd.go

```diff
@@ -539,6 +542,25 @@ func (sc *SPDController) getWorkload(gvr schema.GroupVersionResource, namespace,
 	return workload, nil
 }
 
+func (sc *SPDController) defaultBaselineRatio(workload *unstructured.Unstructured) *float32 {
```

Comment:
for safety reasons, the default value when error appears should be 100 rather than nil (which actually equals 0)

Reply:
and remember to add some comments for why we should return 100